### PR TITLE
Add first pass at portfolio homepage

### PR DESCRIPTION
**Pull request by Kwameoab**
- Add picture of myself to page, scaled and in the center
- Add quick write-up about myself
- Change button and contents that show up when button is clicked on.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,11 +8,17 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Kwame Ampadu-Boateng's Portfolio</h1>
+      <p>Hello! My name is Kwame Ampadu-Boateng. I a currently a rising
```
Comment (ljfranklin) [inline portfolio/src/main/webapp/index.html:13]: Add a newline after `<p>`, makes it easier to see where the tag ends if the end tag is lined up with the start tag. e.g.

```
<p>
  some text.
  some more text
</p>
```

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -8,11 +8,17 @@
...
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Kwame Ampadu-Boateng's Portfolio</h1>
+      <p>Hello! My name is Kwame Ampadu-Boateng. I a currently a rising 
+          sophomore at Vanderbilt University, majoring in Computer Science and 
+          Economics. This is a website I made throught Google's STEP(Student 
+          Training in Engineering Program).</p>
+        <img src="/images/Self.jpg" width="300" height="350"
```
Comment (ljfranklin) [inline portfolio/src/main/webapp/index.html]: Indent the `` tag back one level. All children within a given element should have the same indentation.

**Review by ljfranklin — CHANGES_REQUESTED** [review]

**Review by ljfranklin — CHANGES_REQUESTED** [review]
Also going forward you should add a line break in your commit messages so it doesn't get cut off when displaying in GitHub. The first line should also be a summary. E.g.

```
Add first pass at portfolio homepage

- Add picture of myself to page, scaled and in the center
- Add quick write-up about myself
- Change button and contents that show up when button is clicked on.
```

**Comment by ljfranklin** [comment]
@Kwameoab when you're finished making changes you can click Request next to my name under Reviewers and I'll get a notification to take another look.

**Comment by Kwameoab** [comment]
> @Kwameoab when you're finished making changes you can click Request next to my name under Reviewers and I'll get a notification to take another look.

Thank you so much for telling me that, I was confused on what the next step for me was.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -8,11 +8,20 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Kwame Ampadu-Boateng's Portfolio</h1>
+
+      <p>Hello! My name is Kwame Ampadu-Boateng. I a currently a rising
```
Comment (ljfranklin) [inline portfolio/src/main/webapp/index.html:13]: Nit (short for nitpick): put a line break after the start tag if the contents span multiple lines, e.g. 

```
<p>
    Hello! My name is Kwame Ampadu-Boateng. I a currently a rising
    ...
</p>
```

**Review by ljfranklin — APPROVED** [review]

**Pull request merged**